solar/array.py

```diff
@@ -1,8 +1,11 @@
 from math import pi
 from solar import SolarDay
 import csv
+import os
 
+str = ('python3.7 calculation-angles-cell.py')
```

Comment:
hey, so a problem that might come up with doing it this way is if the command doesn't run on everyone's system the same. (For mine, I had to change this to py -3, for example.) Could we rework this so that it can run everywhere?